Documentation/preparing-devel-environment.md

```diff
@@ -0,0 +1,94 @@
+# Preparing development environment and first rkt build
+This is example configuration for fresh instalation of Ubuntu 16.04 GNOME.
+
+### Required software
+You have to install few software packages to correctly build:
+```sh
+$ sudo apt-get install libssl-dev dh-autoreconf libacl1-dev libtspi-dev build-essential
+```
+See also dependencies [page](https://github.com/coreos/rkt/blob/master/Documentation/dependencies.md).
+
+### We definitely recommend using ccache for kvm flavor developers
+Ccache can save a lot of your time. Once build kernel can be taken from cache, so you don't have to build it over and over. You can get and configure ccache in few easy steps:
+
+```sh
+$ sudo apt-get install ccache
+$ ccache --max-size=10G
+$ sudo ln -s /usr/bin/ccache /usr/local/bin/gcc
+$ sudo ln -s /usr/bin/ccache /usr/local/bin/g++
+```
+We set maximum cache size to 10GB. (Default values are too small to perform kernel compilation caching)
+
+### Installing Go Programming Language for single-user
+```
+$ cd ~/Downloads
+$ curl -O https://storage.googleapis.com/golang/go1.6.1.linux-amd64.tar.gz
+$ tar -xvf go1.6.1.linux-amd64.tar.gz
+$ mv go ~/.local
+```
+
+### Get rkt repo
+I like to keep all my git repos in ~/Repos so..:
+```sh
+$ mkdir -p ~/.local/gopath/src/github.com/coreos
+$ git -C ~/.local/gopath/src/github.com/coreos clone https://github.com/coreos/rkt.git
+$ ln -s ~/.local/gopath/src/github.com/coreos Repos
+```
+### Add GO variables into .bashrc file:
+```sh
+export PATH=~/.local/bin:~/.local/go/bin:$PATH
+export GOPATH=~/.local/gopath
+export GOROOT=~/.local/go
+```
+
+### Building preparation
+rkt always should be build with `LC_ALL=C`.
```

Comment:
AFAIR there is an issue filled for this (but i cant find it) and probably it's assigned to @krnowak (i'm sure that he is aware about this).
I was almost sure that this is tracked in #1122 but I have to correct my memory...

Probably quick fix for this would be adding such `LC_ALL=C` in [this](https://github.com/coreos/rkt/blob/b01a806fc9e23cb4ca1033dd84cb129a92ed5855/tools/depsgen/globcmd.go#L34) line, but i did not tested this.